### PR TITLE
coremark: Add coremark package

### PR DESCRIPTION
**Pull request by gwlim**
Signed-off-by: Lim Guo Wei <limguowei@gmail.com>

Maintainer: Lim Guo Wei
Compile tested: (put here arch, model, OpenWrt version)
Run tested: (AR71xx, WDR4300, 17.01, Run coremark binary expect result obtained)

Description:
CoreMark(r) is an industry-standard benchmark that measures the performance of central processing units (CPU) and embedded microcrontrollers (MCU).

### CODE REVIEW

**Comment by karlp** [comment]
Superseds https://github.com/openwrt/packages/pull/6896

**Comment by karlp** [comment]
I dont' see any reason for them to go and add toolchain options explicitly in the package, we don't do this for anything else.

**Comment by diizzyy** [comment]
We have several packages that allows higher optimization settings
lame, ffmpeg, fdk-aac, openssl, zlib to mention a few

File: utils/coremark/Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=coremark
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/eembc/coremark.git
```
Comment (neheb) [inline utils/coremark/Makefile]: I would use codeload here.

**Comment by BKPepe** [comment]
@gwlim If you are still interested to get it merged, look at neheb comment and rebase this PR against current master. If not, I will close this within a week.

**Comment by diizzyy** [comment]
Closing this due to timeout

**Pull request closed without merging**